ui/pages/[page].js

```diff
@@ -1,16 +1,15 @@
-import { Page, TableOfContents } from '../components';
+import DOMPurify from 'isomorphic-dompurify';
+import { Page, TableOfContents, MarkdownRender } from '../components';
 import { getPages } from '../helpers/api';
-import ReactMarkdown from 'react-markdown';
 
-const StaticPage = ({ pageData }) => {
-  const { content, title, show_table_of_contents: showContents } = pageData;
+const StaticPage = ({ content, showToc, title }) => {
   return (
     <Page title={title}>
       <div className="nhsuk-grid-row">
-        {showContents && <TableOfContents content={content} />}
+        {showToc && <TableOfContents content={content} />}
```

Comment:
Does this still work if the content is html and not markdown?

Reply:
Just pushed a small change to this, to allow it to use a sanitised HTML prop